jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -140,8 +141,18 @@ def vue_unit_conversion(self, *args, **kwargs):
                 self.hub.broadcast(snackbar_message)
 
                 return
+
+        # Uncertainty converted to new flux units
+        temp_uncertainty = self.spectrum.uncertainty.quantity.to(u.Unit(set_flux_unit.unit),
+                                                      equivalencies=u.spectral_density(set_spectral_axis_unit))
```

Comment:
```suggestion
                                                      equivalencies=u.spectral_density(set_spectral_axis_unit))
```

Does this actually work? `u.spectral_density` should be taking an array of dispersion values, not a unit object...

Reply:
`set_spectral_axis_unit` is actually the converted spectral axis, not just the unit.